pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -316,13 +316,24 @@ func interPodAffinityScore(
 	}
 
 	nodescoreList := make(k8sframework.NodeScoreList, len(nodes))
-	errCh := make(chan error, len(nodes))
-	workqueue.ParallelizeUntil(context.TODO(), 16, len(nodes), func(index int) {
+	// the default parallelization worker number is 16.
+	// the whole scoring will fail if one of the processes failed.
+	// so just create a parallelizeContext to control the whole ParallelizeUntil process.
+	// if the parallelizeCancel is invoked, the whole "ParallelizeUntil" goes to the end.
+	// this could avoid extra computation, especially in huge cluster.
+	// and the ParallelizeUntil guarantees only "workerNum" goroutines will be working simultaneously.
+	// so it's enough to allocate workerNum size for errCh.
+	// note that, in such case, errCh should be ge than parallelization number
+	workerNum := 16
```

Comment:
why use this value?

Reply:
16 or workerNum?
16 is the default Parallelization number from volcano.
For "workerNum", I want to express the correlation between size of errCh and Parallelization number. And the min(Parallelization number, len(nodes)) for size of errCh is the least number for main goroutine safety.
But I don't want to make it too complicated, so I did't do min(Parallelization number, len(nodes)) for size of errCh

Reply:
ok, I got it.